### DISCUSSION TITLE
System Requirements Specification (SRS) document

### DISCUSSION THREAD

**Discussion by TemirlanAidarov** [avajs/ava#2977] [Q&A] [answered]
Hello!
I have just found about this project.
I am Computer Science student who takes a course related to Open Software Project.
We have an assignment to write/improve System Requirements Specification (SRS) document for any project. Basically, the SRS is a document that describes the nature of a project, software or application. In simple words, SRS document is a manual of a project provided it is prepared before you kick-start a project/application.

Can I write/improve SRS document for this project?

**Comment by novemberborn** [comment, marked as answer]
@TemirlanAidarov yes, though I see little use for including it in the repository, and I won't have the time to answer many questions with the urgency you may require to complete the assignment.
